stdlib/public/core/ArraySlice.swift

```diff
@@ -1219,7 +1219,7 @@ extension ArraySlice {
   ///
   /// - Parameter body: A closure with an `UnsafeMutableBufferPointer`
   ///   parameter that points to the contiguous storage for the array.
-  ///    If `body` has a return value, that value is also
+  ///    If no such storage exists, it is created. If `body` has a return value, that value is also
```

Comment:
When did this behavior change? Or was the documentation just missing this fact?

Reply:
I believe it was always like this. I could track this line yet in Arrays.swift.gyb. I believe when this comment was added to Array it was forgotten to add it to ArraySlice as well.